docs/src/main/paradox/server/details.md

```diff
@@ -43,9 +43,44 @@ Scala
 Java
 :   @@snip[ExceptionGreeterServiceImpl](/interop-tests/src/test/java/example/myapp/helloworld/grpc/ExceptionGreeterServiceImpl.java) { #streaming }
 
+## Rich error model
+Beyond status codes you can also use the [Rich error model](https://www.grpc.io/docs/guides/error/#richer-error-model). Currently there is no native support for this concept in Akka gRPC. However you can use the following manual approach.
+
+Add the following dependency to receive required classes (that are based on the [common protobuf](https://cloud.google.com/apis/design/errors#error_model)):
+
+`sbt`
+:   @@@vars
+```sbt
+libraryDependencies += "io.grpc" % "grpc-protobuf" % "1.42.1"
```

Comment:
We have a `grpc.version` paradox property (https://github.com/akka/akka-grpc/blob/main/build.sbt#L188), perhaps that could be used here so the version gets automatically updated?